js/app.js

```diff
@@ -55,18 +55,89 @@ function isDupe(picture) {
     return true;
   } else if (picture === Picture.lastSet[2]) {
     return true;
+  } else if (picture === Picture.currentSet[0]) {
+    return true;
+  } else if (picture === Picture.currentSet[1]) {
+    return true;
+  } else if (picture === Picture.currentSet[2]) {
+    return true;
+  } else {
+    return false;
```

Comment:
Just FYI: You can simplify all these if/else statements because you are returning `true` when a condition evaluates to `true`. All of this is equivalent to:
```javascript
return picture === Picture.lastSet[0] || picture === Picture.lastSet[1] || picture === Picture.lastSet[2] || picture === Picture.currentSet[0] || picture === Picture.currentSet[1] || picture === Picture.currentSet[2]
```